src/libs/config.js

```diff
@@ -11,8 +11,7 @@ export const getConfig = () => {
 export const isAnvil = () => (window.location.hostname === 'anvil.terra.bio') || getConfig().isAnvil
 export const isAnalysisTabVisible = () => getConfig().isAnalysisTabVisible
 export const isCromwellAppVisible = () => getConfig().isCromwellAppVisible
-// configOverridesStore.set({ isDataBrowserVisible: true }) in browser console to enable
-export const isDataBrowserVisible = () => getConfig().isDataBrowserVisible
+export const isDataBrowserFrontPage = () => false
```

Comment:
If there isn't one already, can you make a ticket for the work to show the front page banner, or to remove it if we decide not to include it after all?